components/channel_header_dropdown/index.ts

```diff
@@ -89,7 +91,7 @@ const mobileMapStateToProps = (state) => {
         channel,
         teammateId,
         teammateIsBot,
-        teammateStatus: getTeammateStatus(state),
+        teammateStatus: getTeammateStatus(state) ?? undefined,
```

Comment:
the selector can return undefined from it so we dont have to do this fallback here.
```
if (!teammateId) {
            return undefined;
        }
```